package.json

```diff
@@ -1,13 +1,14 @@
 {
   "name": "sandify",
   "homepage": "https://sandify.org",
-  "version": "0.1.7",
+  "version": "0.1.8",
   "private": true,
   "dependencies": {
     "@reduxjs/toolkit": "^1.2.5",
+    "node-sass": "^4.13.1",
     "rc-slider": "^8.6.3",
     "react": "^16.13.0",
-    "react-bootstrap": "^0.31.0",
+    "react-bootstrap": "^1.0.0-beta.17",
```

Comment:
Should we be worried that this is marked beta?

Reply:
I don't think so. It's a late beta, our UI is pretty minimal, and I didn't run into any issues. It also gets us to BS4, which is good for a number of reasons. That said, lmk if you see anything weird or feel this is risky.